api/openapi-spec/swagger.json

```diff
@@ -3601,6 +3608,28 @@
      }
     }
    },
+   "v1.Interface": {
+    "required": [
+     "name"
+    ],
+    "properties": {
+     "bridge": {
+      "$ref": "#/definitions/v1.InterfaceBridge"
+     },
+     "name": {
+      "description": "Logical name of the interface as well as a reference to the associated networks\nMust match the Name of a Network",
+      "type": "string"
+     }
+    }
+   },
+   "v1.InterfaceBridge": {
+    "properties": {
+     "delegateIp": {
```

Comment:
Can we introduce `delegateIp` once it is used?

Reply:
Yes, I'll remove it for now.